cmd/apiserver/apiserver.go

```diff
@@ -27,8 +27,6 @@ import (
 	"k8s.io/kubernetes/pkg/util/logs"
 
 	"github.com/kubernetes-incubator/service-catalog/cmd/apiserver/app/server"
-	// install our API groups
-	_ "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/install"
```

Comment:
wow, really? This blows my mind. I don't even know what to believe anymore. Where does anything get installed?

😖